### PR TITLE
winetricks: update

### PR DESCRIPTION
**Pull request by abbradar**
Updates and cleans up the builder. Patch is still needed [1](https://code.google.com/p/winetricks/issues/detail?id=477). cc @the-kenny (I'll push this in a day, but maybe you have something to add?)

### CODE REVIEW

File: pkgs/misc/emulators/wine/winetricks.nix [hunk 1]
```diff
@@ -1,25 +1,31 @@
-{ stdenv, fetchsvn, wine, perl, which, coreutils, zenity, curl
+{ stdenv, fetchgit, wine, perl, which, coreutils, zenity, curl
 , cabextract, unzip, p7zip, gnused, gnugrep, bash } :
 
 stdenv.mkDerivation rec {
-  rev = "1199";
-  name = "winetricks-${rev}";
+  name = "winetricks";
```
Comment (jagajaga) [inline pkgs/misc/emulators/wine/winetricks.nix]: Version (date)?

Comment (abbradar) [inline pkgs/misc/emulators/wine/winetricks.nix]: Thanks for reminding! I've used to left vcs-sourced packages without a version earlier but this is against our packaging style.

**Comment by abbradar** [comment]
Upstream merged the patch, good -- removed it, too.

**Comment by abbradar** [comment]
Waiting for upstream to comment on another patch.

**Pull request merged**